spec/services/create_induction_tutor_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe CreateInductionTutor do
+  let(:school) { create(:school) }
+  let(:name) { Faker::Name.name }
+  let(:email) { Faker::Internet.email }
+
+  describe ".call" do
+    it "creates a user with an induction coordinator profile" do
+      expect {
+        CreateInductionTutor.call(school: school, email: email, full_name: name)
+      }.to change { InductionCoordinatorProfile.count }.by(1)
+        .and change { User.count }.by(1)
+    end
+
+    it "emails the new induction tutor" do
+      allow(SchoolMailer).to receive(:nomination_confirmation_email).and_call_original
+
+      service = CreateInductionTutor.new(school: school, email: email, full_name: name)
+      service.call
+
+      expect(SchoolMailer).to have_received(:nomination_confirmation_email)
+        .with(user: service.profile.user, school: school, start_url: service.start_url)
+    end
```

Comment:
Rather than using `service.profile.user` (is that the only reason the service has a `profile` field?), I would be tempted to add `new_user = User.find_by(email: email)` after the `call` and before the `expect`, and check that that `user` is what the mailer received. 

Emails are unique per user, and it will mean we don't need to store the profile we just created on the service instance.

Reply:
Yes could do